docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,9 +1,11 @@
 = Devfile
 :description: Software Defined Development Environments
 
-Devfiles describe and define a cloud native development environment.
+Devfiles are YAML files that describe and define an environment in Che, including the source code and development components, like IDE tools and application runtimes. Che uses these devfiles to make a cloud workspace composed of multiple containers.
```

Comment:
Devfiles can be used outside of Che in other tools,such as in Odo, and eventually the OpenShift console, among other tools. 

Personally I prefer the generic "cloud native development environment" since it's not specific to any one tool or platform.

Reply:
Thanks, John!

Reply:
+1 for using cloud native development environment and making the description generic and not restricting that to Che.

Reply:
@jc-berger, hyphenate compound adjectives.
```suggestion
Devfiles are YAML files that describe and define cloud-native development environments, including source code and development components, like IDE tools and application runtimes. Various applications, including link:https://www.eclipse.org/che/[Eclipse Che], use devfiles to create cloud workspaces composed of multiple containers.
```